frontend/src/scenes/pipeline/types.ts

```diff
@@ -12,6 +12,7 @@ export enum PipelineBackend {
     Plugin = 'plugin',
     HogFunction = 'hog_function',
     ManagedSource = 'managed_source',
+    SelfManagedSource = 'data_warehouse',
```

Comment:
I still think the key here needs updating to something other than `data_warehouse`